### PR TITLE
send fcm upstream

### PR DESCRIPTION
**Pull request by n0mer**

### CODE REVIEW

**Comment by Hirbod** [comment]
The code has a fixed firebase-domain. this isn't useable

**Comment by soumak77** [comment]
closing for cleanup, please resubmit PR if changes are still desired

**Comment by n0mer** [comment]
@Hirbod so now it is possible to send fcm upstream messages? Can you please provide URL for exact project, because i do not completely understand what firebase-domain means.

Thanks in advance

File: src/android/FirebasePlugin.java [hunk 1]
```diff
@@ -537,4 +542,31 @@ public void run() {
...
+                    Iterator<?> keys = data.keys();
+
+                    while( keys.hasNext() ){
+                        String key = (String)keys.next();
+                        String value = data.getString(key);
+                        map.put(key, value);
+                    }
+
+                    FirebaseMessaging fm = FirebaseMessaging.getInstance();
+                    fm.send(new RemoteMessage.Builder("78391781623@gcm.googleapis.com")
```
Comment (Hirbod) [inline src/android/FirebasePlugin.java]: what is this?

**Pull request closed without merging**